lib/components/viewers/trip-viewer.js

```diff
@@ -87,89 +102,94 @@ class TripViewer extends Component {
                     }}
                   />
                 )}
-              </div>
+              </RouteName>
 
               {/* Wheelchair/bike accessibility badges, if applicable */}
-              <h4>
-                {tripData.wheelchairAccessible === 1 && (
-                  <BsLabel bsStyle="primary">
-                    <IconWithText Icon={Wheelchair}>
-                      <FormattedMessage id="components.TripViewer.accessible" />
-                    </IconWithText>
-                  </BsLabel>
-                )}
-                <SpanWithSpace margin={0.25} />
-                {tripData.bikesAllowed === 1 && (
-                  <BsLabel bsStyle="success">
-                    <IconWithText Icon={Bicycle}>
-                      <FormattedMessage id="components.TripViewer.bicyclesAllowed" />
-                    </IconWithText>
-                  </BsLabel>
-                )}
-              </h4>
+              {(tripData.wheelchairAccessible === 1 ||
+                tripData.bikesAllowed === 1) && (
+                <div>
+                  {tripData.wheelchairAccessible === 1 && (
+                    <BsLabel bsStyle="primary">
+                      <IconWithText Icon={Wheelchair}>
+                        <FormattedMessage id="components.TripViewer.accessible" />
+                      </IconWithText>
```

Comment:
Note (TODO): the labeling of the wheelchair/bike badges will need to be redone. The labeling of these badges is currently incomplete for screen readers.